runtime/expr/dot.go

```diff
@@ -75,6 +79,16 @@ func (d *DotExpr) Eval(ectx Context, this *zed.Value) *zed.Value {
 	return ectx.NewValue(recType.Columns[idx].Type, field)
 }
 
+func (d *DotExpr) evalTypeOfType(ectx Context, vb zcode.Bytes) *zed.Value {
+	typ, _ := d.zctx.DecodeTypeValue(vb)
```

Comment:
Nit: Call this `b`, `bytes`, or `zb`.
```suggestion
func (d *DotExpr) evalTypeOfType(ectx Context, b zcode.Bytes) *zed.Value {
	typ, _ := d.zctx.DecodeTypeValue(b)
```

Reply:
ok